### PR TITLE
utils: add JSON writer

### PR DESCRIPTION
**Pull request by rerowep**
* Adds JsonWriter class to write json data well formatted to a file.

Co-Authored-by: Peter Weber <peter.weber@rero.ch>

## Code review check list

- [ ] Commit message template compliance.
- [ ] Commit message without typos.
- [ ] File names.
- [ ] Functions names.
- [ ] Functions docstrings.
- [ ] Unnecessary commited files?
- [ ] Cypress tests successful?

### CODE REVIEW

File: rero_ils/modules/utils.py [hunk 1]
```diff
@@ -990,3 +990,45 @@ def requests_retry_session(retries=3, backoff_factor=0.3,
...
+
+class JsonWriter(object):
+    """Json Writer."""
+
+    count = 0
+
+    def __init__(self, filename, indent=2):
+        """Constructor.
+
+        :params filename: file name of write file.
```
Comment (iGormilhit) [inline rero_ils/modules/utils.py:1003]: File name of the file to be written? File name of the written file? Something else?

File: rero_ils/modules/utils.py [hunk 2]
```diff
@@ -990,3 +990,45 @@ def requests_retry_session(retries=3, backoff_factor=0.3,
...
+        :param indent: indention.
```
Comment (iGormilhit) [inline rero_ils/modules/utils.py:1004]:
```suggestion
        :param indent: indentation.
```

File: rero_ils/modules/utils.py [hunk 3]
```diff
@@ -990,3 +990,45 @@ def requests_retry_session(retries=3, backoff_factor=0.3,
...
+        """Destructor."""
+        if self.file_handle:
+            self.file_handle.write('\n]')
+            self.file_handle.close()
+            self.file_handle = None
+
+    def write(self, data):
+        """Write data to file.
+
+        :param data: json data to write.
```
Comment (iGormilhit) [inline rero_ils/modules/utils.py:1020]:
```suggestion
        :param data: JSON data to write into the file.
```

**Review by iGormilhit — CHANGES_REQUESTED** [review]
Commit message:

```
CLI: add a JSON writer

* Adds a `JsonWriter` class to write well formatted JSON data into a file.
```

**Review by zannkukai — APPROVED** [review]

**Review by iGormilhit — APPROVED** [review]

**Review by lauren-d — APPROVED** [review]

**Pull request merged**